package.json

```diff
@@ -98,18 +99,18 @@
     "jasmine-core": "^2.6.1",
     "json-loader": "^0.5.4",
     "karma": "^1.7.0",
-    "@types/log4js": "0.0.33",
     "karma-chrome-launcher": "^2.1.1",
     "karma-coverage-istanbul-reporter": "^1.2.1",
     "karma-jasmine": "^1.1.0",
     "karma-mocha-reporter": "^2.2.3",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^2.0.3",
-    "node-sass": "^4.5.2",
+    "node-sass": "^4.11.0",
     "raw-loader": "^0.5.1",
     "rimraf": "2.6.1",
     "rxjs": "^5.3.1",
     "sass-loader": "^6.0.5",
+    "snapjs-aws": "git+https://37327d5496e19922f3042f023495176af24ba924:x-oauth-basic@github.com/SnapMobileIO/SnapJS-Aws.git",
```

Comment:
Glad you got this hooked up

Reply:
@dukegreene this was not the problem, if thats what you're referring to. i mean this was a problem, but not what i was dealing with yesterday.